// @westpac/wbc/dist/wbc.cjs.prod.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: !0
});

var OVERRIDES = function(TOKENS) {
  return {};
}, bodyFont = "some family", brandFont = '"some-other-family"', SPACING = function(i, minor) {
  var unit = arguments.length > 2 && void 0 !== arguments[2] ? arguments[2] : "rem";
  return (6 * i - (minor && 0 !== i ? 3 : 0)) / 16 + (unit && i > 0 ? unit : 0);
}, COLORS = {
  tints: {
    color15: "#FFF2F2",
    color110: "#FFE6E6",
    color120: "#FFCCCC",
    color130: "#FFB3B3",
    color140: "#FF9999",
    color150: "#FF8080",
    color160: "#FF6666",
    color170: "#FF4D4D",
    color180: "#FF3333",
    color190: "#FF1919",
    color25: "#FFFFF2",
    color210: "#FFFFE6",
    color220: "#FFFFCC",
    color230: "#FFFFB3",
    color240: "#FFFF99",
    color250: "#FFFF80",
    color260: "#FFFF66",
    color270: "#FFFF4D",
    color280: "#FFFF33",
    color290: "#FFFF19",
    color35: "#FFFFFF",
    color310: "#FFFFFF",
    color320: "#FFFFFF",
    color330: "#FFFFFF",
    color340: "#FFFFFF",
    color350: "#FFFFFF",
    color360: "#FFFFFF",
    color370: "#FFFFFF",
    color380: "#FFFFFF",
    color390: "#FFFFFF"
  },
  color1: "#ff0000",
  color2: "#ffff00",
  color3: "#ffffff"
}, LAYOUT = {
  breakpoints: {
    sm: 100,
    md: 200,
    lg: 300
  }
}, TYPE = {
  files: {
    "": [ {
      "@font-face": {
        fontFamily: '"some-other-family"',
        src: 'url("_PATH_some-other-family-light.woff2") format("woff2"), url("_PATH_some-other-family-light.woff") format("woff")',
        fontWeight: "300",
        fontStyle: "normal"
      }
    }, {
      "@font-face": {
        fontFamily: '"some-other-family"',
        src: 'url("_PATH_some-other-family-regular.woff2") format("woff2"), url("_PATH_some-other-family-regular.woff") format("woff")',
        fontWeight: "400",
        fontStyle: "normal"
      }
    }, {
      "@font-face": {
        fontFamily: '"some-other-family"',
        src: 'url("_PATH_some-other-family-bold.woff2") format("woff2"), url("_PATH_some-other-family-bold.woff") format("woff")',
        fontWeight: "700",
        fontStyle: "normal"
      }
    } ]
  },
  bodyFont: {
    fontFamily: bodyFont,
    100: {
      fontFamily: bodyFont,
      fontWeight: 100
    },
    200: {
      fontFamily: bodyFont,
      fontWeight: 200
    },
    300: {
      fontFamily: bodyFont,
      fontWeight: 300
    },
    400: {
      fontFamily: bodyFont,
      fontWeight: 400
    },
    500: {
      fontFamily: bodyFont,
      fontWeight: 500
    },
    600: {
      fontFamily: bodyFont,
      fontWeight: 600
    },
    700: {
      fontFamily: bodyFont,
      fontWeight: 700
    },
    800: {
      fontFamily: bodyFont,
      fontWeight: 800
    },
    900: {
      fontFamily: bodyFont,
      fontWeight: 900
    }
  },
  brandFont: {
    fontFamily: brandFont,
    100: {
      fontFamily: brandFont,
      fontWeight: 300
    },
    200: {
      fontFamily: brandFont,
      fontWeight: 300
    },
    300: {
      fontFamily: brandFont,
      fontWeight: 300
    },
    400: {
      fontFamily: brandFont,
      fontWeight: 400
    },
    500: {
      fontFamily: brandFont,
      fontWeight: 400
    },
    600: {
      fontFamily: brandFont,
      fontWeight: 400
    },
    700: {
      fontFamily: brandFont,
      fontWeight: 700
    },
    800: {
      fontFamily: brandFont,
      fontWeight: 700
    },
    900: {
      fontFamily: brandFont,
      fontWeight: 700
    }
  }
}, PACKS = {
  subdir: {
    subPack1: {
      fontSize: "3.375rem",
      lineHeight: 1.2,
      fontFamily: "some family"
    },
    subPack2: {
      fontSize: "4rem",
      color: "blue"
    },
    subPack3: {
      fontSize: "3.375px",
      lineHeight: 2
    }
  },
  pack1: {
    fontSize: [ "1rem", "1.125rem" ],
    lineHeight: 1.4
  },
  pack2: {
    color: "#ff0000",
    ":hover": {
      color: "#ffff00"
    }
  }
}, BRAND = "WBC", OVERRIDES$1 = OVERRIDES(), index = {
  SPACING: SPACING,
  COLORS: COLORS,
  LAYOUT: LAYOUT,
  TYPE: TYPE,
  PACKS: PACKS,
  BRAND: BRAND,
  OVERRIDES: OVERRIDES$1
};

exports.BRAND = BRAND, exports.COLORS = COLORS, exports.LAYOUT = LAYOUT, exports.OVERRIDES = OVERRIDES$1,
exports.PACKS = PACKS, exports.SPACING = SPACING, exports.TYPE = TYPE, exports.default = index;
